pipelined/bevy_ui2/src/ui_node.rs

```diff
@@ -251,7 +251,29 @@ impl Default for FlexWrap {
     }
 }
 
-#[derive(Default, Copy, Clone, Debug)]
+#[derive(Component, Default, Copy, Clone, Debug, Reflect)]
+#[reflect(Component)]
 pub struct CalculatedSize {
     pub size: Size,
 }
+
+#[derive(Component, Default, Copy, Clone, Debug, Reflect)]
+#[reflect(Component)]
+pub struct Color(pub bevy_render2::color::Color);
+
+impl From<bevy_render2::color::Color> for Color {
+    fn from(color: bevy_render2::color::Color) -> Self {
+        Self(color)
+    }
+}
+
+// NOTE: Option<Handle<Image>> is reflected as a value, so this derive(Reflect) requires Handle<T> to derive Serialize and Deserialize
+#[derive(Component, Default, Clone, Debug, Reflect)]
+#[reflect(Component)]
+pub struct Image(pub Option<Handle<bevy_render2::texture::Image>>);
+
+impl From<Handle<bevy_render2::texture::Image>> for Image {
+    fn from(handle: Handle<bevy_render2::texture::Image>) -> Self {
+        Self(Some(handle))
+    }
+}
```

Comment:
The names of these new components are horrible since they alias the types they contain. Any suggestions?

Reply:
Prepend `Ui` to the names? So like `UiColor`, `UiImage` etc.

Reply:
Done in c8975e5